content/news/christmas-2017.md

```diff
@@ -0,0 +1,16 @@
+
+This year, we have to start with some really unfortunate news. Santa's sleight carrying the next OpenRA release was shot down by an AA gun!
```

Comment:
How about "We are sad to report that Santa's sleigh was shot down by an Allied AA gun, and our planned Christmas Playtest was lost overboard!  We are working hard towards recovering the new build in the coming weeks."?